js/views/userPage/Home.js

```diff
@@ -1,15 +1,29 @@
 import $ from 'jquery';
+import _ from 'underscore';
 import loadTemplate from '../../utils/loadTemplate';
 import { clipboard } from 'electron';
 import BaseVw from '../baseVw';
+import app from '../../app';
+import UserCard from '../userCard';
+import { launchModeratorDetailsModal } from '../../utils/modalManager';
+import { openSimpleMessage } from '../modals/SimpleMessage';
 
 export default class extends BaseVw {
   constructor(options = {}) {
     super(options);
     this.options = options;
 
     this.ownPage = options.ownPage;
-    this.currentModerator = false; // TODO set if this user is one of the viewer's moderators
+
+    this.userCard = new UserCard({ model: this.model, hideControls: true });
+
+    this.settings = app.settings.clone();
+
+    this.listenTo(this.settings, 'sync', () => {
+      app.settings.set(this.settings.toJSON());
+      this.$addModeratorLbl.toggleClass('hide', this.ownMod);
```

Comment:
Same as the comment on the user card... if we move the adjustment of the state of the two buttons to the change:storeModerators on app.settings then we'll be good if some other area initiated the change.